docs/3.0rc/develop/write-flows.mdx

```diff
@@ -144,6 +129,36 @@ When you run this flow, you'll see the following output:
 It was Friday on 2021-01-01T02:00:19.180906
 ```
 
+---
+
+You may pass also a `dict` representation of a `BaseModel` parameter and it will be coerced:
+```python
+from prefect import flow
+from pydantic import BaseModel
+
+
+class Model(BaseModel):
+    a: int
+    b: str
+
+
+@flow
+def flow_that_validates_parameters(model: Model): ...
+
+if __name__ == "__main__":
+    try:
+        flow_that_validates_parameters(
+            model={"a": "WRONG", "b": "fine"}
+        )
+    except Exception as e:
```

Comment:
I don't think I understand the `try/except/print` here - wouldn't the error get raised and displayed normal-style without this?

Reply:
:doh: yeah you're totally right, just a lapse on my end

updated in [ba904c1](https://github.com/PrefectHQ/prefect/pull/15146/commits/ba904c168a824e445013a011067f080ed60e39fd)